### ISSUE TITLE
Cannot Track Fish

### ISSUE THREAD

**Issue by mrgrim** [JereKuusela/valheim-esp#6]
I believe this is due to fish not having an instance of the BaseAI class. I don't have a proper development environment up to experiment with and try to fix this on my own. I've never setup a C# environment before or used visual studio. I was able to get ILSpy up and at least track this much down. I'm hoping this is a simple enough addition that it won't consume too much of your time if you're willing to add support.

Thanks!

**Comment by JereKuusela** [comment]
Yes, currently the beam is only done for BaseAI. I could add it to fish too.
